File: python/example_1b.py
UPPER_RANGE = 32767

def main():
    k = 0
    for i in range(1, UPPER_RANGE):
        if (i%3 == 2 and i%5==3 and i%7==4):
            k+=1
            print("the "+str(k)+" number is "+str(i))

    print("There were "+str(k)+" target values")

main()

File: ruby/example_1b.rb
class Example1b
    UPPER_LIMIT = 32767
    def self.run 
        k = 1
        for i in (1..UPPER_LIMIT)
            if i % 3 == 2 &&
               i % 5 == 3 &&
               i % 7 == 4
              puts "The #{k} number is #{i}"
              k+=1
            end
        end
        puts "The total number of target values is #{k-1}"
    end
end

Example1b.run
